scripts/aws/setupdb.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Export settings required to run psql non-interactively
+export PGHOST=$(cat /etc/mmw.d/env/MMW_DB_HOST)
+export PGDATABASE=$(cat /etc/mmw.d/env/MMW_DB_NAME)
+export PGUSER=$(cat /etc/mmw.d/env/MMW_DB_USER)
+export PGPASSWORD=$(cat /etc/mmw.d/env/MMW_DB_PASSWORD)
```

Comment:
Kind of begs the question... why not just use these env var names instead? It would be nice to be able to run `envdir /etc/mmw.d/env psql`.